workflow-examples/src/test/java/com/redhat/parodos/examples/integration/utils/ExamplesUtils.java

```diff
@@ -167,7 +167,7 @@ public static ProjectResponseDTO getProjectByNameAndDescription(List<ProjectResp
 			String projectName, String projectDescription) {
 		return projects.stream()
 				.filter(prj -> projectName.equals(prj.getName()) && projectDescription.equals(prj.getDescription())
-						&& prj.getUsername() == null && !Strings.isNullOrEmpty(prj.getId()))
+						&& prj.getUsername() == null && prj.getId() == null)
```

Comment:
not sure if it's ok.  `prj.getId` shouldn't be null, no?

Reply:
Good catch. Should be the reverse condition.